test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js

```diff
@@ -186,6 +255,49 @@ export default class EnrollOktaVerifyPageObject extends BasePageObject {
     return this.form.el.find(DOWNLOAD_OV_LINK_CLASS).getAttribute('href');
   }
 
+  getAppStoreHref() {
+    if (userVariables.gen3) {
+      return this.form.el.find(APP_STORE_LINK_CLASS).getAttribute('href');
+    }
+    return this.form.el.find(APP_STORE_LOGO_CLASS).getAttribute('href');
+  }
+
+  getOVSetupHref() {
+    return this.form.el.find(OV_SETUP_LINK_CLASS).getAttribute('href');
+  }
+
+  async clickBackToSameOVDevice() {
+    await this.form.clickElement(BACK_LINK_CLASS);
+  }
```

Comment:
If this function is only called once in `clickBackLink()`, why can't we just move this one line directly into `clickBackLink()`?

Reply:
please see https://github.com/okta/okta-signin-widget/pull/3526/commits/081366e9ba882712767a9b67c1ae4b299a7cdbfa